Enzyme/enzyme_tut.jl

```diff
@@ -484,7 +484,7 @@ A reverse-mode AD algorithm would often:
 
 The reason for the added complexity of reverse-mode is two-fold:
 * First, one must execute the program backwards. For programs with loops, branching, or nontrivial control flow, setting up this infrastructure may be non-trivial. 
-* Second, unlike forward mode where the derivative is immediately available upon executing the shadow instruction, in reverse  mode we add up all the partial derivatives. This is because if a variable is used multiple times, we can get multiple terms to the derivative from the partials of each of its users.
+* Second, derivatives wrt input variables are accumulated (`+=`), rather than assigned (like forward-mode). This is because the derivative wrt to a variable that gets used multiple times is the sum of derivative contributions from each of its users.
```

Comment:
Maybe expand wrt?

Also the like forward mode in parentheses is ambiguous and could mean that it is accumulated not assigned (like forward mode) meaning forward mode is accumulated not assigned or could mean forward mode assigns